docsrc/getting_started/jetpack.rst

```diff
@@ -101,9 +101,9 @@ Install pip and required python packages:
 Build and Install torch_tensorrt wheel file
 
 
-Since torch_tensorrt version has dependencies on torch version. torch version supported by JetPack6.1 is from DLFW 24.08/24.09(torch 2.5.0).
+Since torch_tensorrt version has dependencies on torch version. torch version supported by JetPack6.2 is from DLFW 24.08/24.09(torch 2.6.0).
```

Comment:
@apbose can you update this with appropriate numbers? also replace `DLFW` with `NVIDIA NGC`